api/v1alpha1/envoyextensionypolicy_types.go

```diff
@@ -0,0 +1,120 @@
+// Copyright Envoy Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	gwapiv1a2 "sigs.k8s.io/gateway-api/apis/v1alpha2"
+)
+
+const (
+	// KindEnvoyExtensionPolicy is the name of the EnvoyExtensionPolicy kind.
+	KindEnvoyExtensionPolicy = "EnvoyExtensionPolicy"
+)
+
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:shortName=sp
```

Comment:
This `shortName` is the same as the `shortName` defined for the `SecurityPolicy` resource. 

Consider removing the short name, or changing it to `ep`.